src/components/UI/JSONSchemaForm/BaseInputTemplate/index.tsx

```diff
@@ -25,9 +25,16 @@ const BaseInputTemplate: React.FC<WidgetProps> = ({
     }
   };
 
+  const handleSuggestionSelect = (e: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    suggestion: any;
```

Comment:
Would it be possible to type this as a string instead of `any`?

Reply:
good catch, fixed